hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/TableResolverImpl.java

```diff
@@ -189,6 +207,21 @@ private Table toTable(Mapping mapping) {
         );
     }
 
+    private Table toTable(View view) {
+        List<TableField> tableFields = new ArrayList<>(view.viewColumnNames().size());
+        for (int i = 0; i < view.viewColumnNames().size(); ++i) {
```

Comment:
Is it intentional to keep invariants `view.viewColumnNames()` and `view.viewColumnNames().size()` inside loop?

Reply:
They are supposed to have same sizes, so, yes.